app/src/main/java/org/sil/gatherwords/room/WordDao.java

```diff
@@ -12,28 +13,28 @@
  */
 @Dao
 public interface WordDao {
-	// SELECT
-	@Query("SELECT * FROM word")
-	List<Word> getAll();
```

Comment:
Are you sure you'll never need this?

Reply:
Highly likely.  This has no filter by session.  If needed, we can readd this one liner.  In the mean time, it would just be a candidate for bit rot.